config/functions/cron.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+/**
+ * Cron config that gives you an opportunity
+ * to run scheduled jobs.
+ *
+ * The cron format consists of:
+ * [SECOND (optional)] [MINUTE] [HOUR] [DAY OF MONTH] [MONTH OF YEAR] [DAY OF WEEK]
+ *
+ * See more details here: https://strapi.io/documentation/v3.x/concepts/configurations.html#cron-tasks
+ */
+
+module.exports = {
+  /**
+   * Simple example.
+   * Every monday at 1am.
+   */
+  // '0 1 * * 1': () => {
+  //
+  // }
+};
```

Comment:
Why we have a `cron` config in this service?

Reply:
@chinzou  default STRAPI file.

Reply:
if it's safe to delete it, then we should do so as long as we're not using it, to keep our code clean :)